core/extension/gdextension.cpp

```diff
@@ -686,7 +721,18 @@ GDExtensionInterfaceFunctionPtr GDExtension::get_interface_function(const String
 }
 
 Error GDExtension::open_library(const String &p_path, const String &p_entry_symbol) {
+	Vector<SharedObject> dependencies;
+	return open_library_with_dependencies(p_path, p_entry_symbol, dependencies);
+}
+
+Error GDExtension::open_library_with_dependencies(const String &p_path, const String &p_entry_symbol, const Vector<SharedObject> &p_dependencies) {
```

Comment:
I assume this new method was added to avoid changing the signature of `GDExtension::open_library()` because it's exposed?

This actually reminds me that we decided at a recent GDExtension meeting to un-expose this method - I'll make a PR for that soon. :-)

Reply:
Here's the PR https://github.com/godotengine/godot/pull/88418 to unbind that method

Once it's no longer exposed, I think it would be fine to add the new arguments directly on `GDExtension::open_library()`

Reply:
Updated!